Frends.IMAP.ReadEmail/README.md

```diff
@@ -1,50 +1,20 @@
 # Frends.IMAP.ReadEmail
 
+Frends Task for reading emails using IMAP protocol.
+
 [![Frends.IMAP.ReadEmail Main](https://github.com/FrendsPlatform/Frends.IMAP/actions/workflows/ReadEmail_main.yml/badge.svg)](https://github.com/FrendsPlatform/Frends.IMAP/actions/workflows/ReadEmail_main.yml)
 ![MyGet](https://img.shields.io/myget/frends-tasks/v/Frends.IMAP.ReadEmail?label=NuGet)
 ![GitHub](https://img.shields.io/github/license/FrendsPlatform/Frends.IMAP?label=License)
 ![Coverage](https://app-github-custom-badges.azurewebsites.net/Badge?key=FrendsPlatform/Frends.IMAP/Frends.IMAP.ReadEmail|main)
 
-Task for reading emails using IMAP protocol.
 
 ## Installing
 
 You can install the Task via frends UI Task View or you can find the NuGet package from the following NuGet feed
 https://www.myget.org/F/frends-tasks/api/v2.
 
```

Comment:
I think you are missing the "Building" header and contents.

Reply:
Building is in the line 16, just under the installing